web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -125,7 +127,7 @@
   transition:fade={{ duration: haveFadeTransition ? 150 : 0 }}
   class="flex h-full select-none place-content-center place-items-center"
 >
-  {#await loadAssetData({ loadOriginal: $alwaysLoadOriginalFile ? isWebCompatibleImage(asset) : false })}
```

Comment:
The issue now occurs again. I would just do `$: isWebCompatible = isWebCompatibleImage(asset);` and use that here

Reply:
Really ? I can't reproduce this issue on firefox and chrome.

>  I would just do $: isWebCompatible = isWebCompatibleImage(asset);

You mean without `$alwaysLoadOriginalFile` ? That would be a regression, it was implemented because some users want to always load the original file like on mobile.

Reply:
It is fixed. My bad, I still had an old version

Reply:
Ahah no worries, thank you for your feedback, it's much better now